be/src/runtime/mem_tracker.h

```diff
@@ -112,7 +113,11 @@ class MemTracker {
     static std::shared_ptr<MemTracker> get_temporary_mem_tracker(const std::string& label);
 
     Status check_sys_mem_info(int64_t bytes) {
```

Comment:
is bytes still virtual memory?

Reply:
yep, so it is still possible to predict OOM too early, when this virtual memory is not used.